execution_engine/src/core/engine_state/mod.rs

```diff
@@ -330,6 +330,8 @@ where
             CLValue::from_t(upgrade_config.chainspec_registry().clone())
                 .map_err(|error| Error::Bytesrepr(error.to_string()))?;
 
+        // We write the checksums of the chainspec settings to global state
+        // allowing verification of the reported chainspec data via the RPC.
```

Comment:
// allowing verification of the chainspec data reported via the RPC.

Reply:
Applied suggestion